package.json

```diff
@@ -245,7 +245,7 @@
     "lru-cache": "^4.1.3",
     "request": "^2.86.0",
     "request-promise-native": "^1.0.5",
-    "vscode-languageclient": "^4.4.0",
+    "vscode-languageclient": "5.3.0-next.5",
```

Comment:
Doesn't the `next` suffix mean you have to be on the dev channel for vscode?

Reply:
No, the vscode version is constrained from the `engines.vscode` field from what I can tell: I tried bumping it to 1.35 today and I got an error when installing it with my older version of vscode. `vscode-languageclient` is just the library for connecting vscode extension APIs to LSP